apps/web/src/pages/CommunityInterests/form.ts

```diff
@@ -155,6 +173,16 @@ export function formValuesToApiUpdateInput(
     trainingInterest: parseMaybeStringToBoolean(formValues.trainingInterest),
     additionalInformation: formValues.additionalInformation,
     interestInDevelopmentPrograms,
+
+    // finance-only fields
+    financeIsChief: formValues.financeIsChief,
+    financeAdditionalDuties: formValues.financeAdditionalDuties
+      ? (formValues.financeAdditionalDuties as FinanceChiefDuty[])
+      : null,
+    financeOtherRoles: formValues.financeOtherRoles
+      ? (formValues.financeOtherRoles as FinanceChiefRole[])
+      : null,
```

Comment:
Instead of using `as` you should be able to create a util that will convert the string to an enum. For example:

```ts
export function stringToEnumLocation(
  selection: string,
): WorkRegion | undefined {
  if (Object.values(WorkRegion).includes(selection as WorkRegion)) {
    return selection as WorkRegion;
  }
  return undefined;
}
```

```ts
      languageAbility: !isEmpty(data.languageAbility)
        ? stringToEnumLanguage(data.languageAbility)
        : undefined,
```

Reply:
Good idea!  I'd like to see if we can make a generic version of this some time but for now I just copied the pattern you suggested.
[type converter for enums](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13011/commits/2ea0164609b7773997d5876069c1cb0b1c95b6c9)